src/js/components/SharedComponents/SubmissionWarningBanner.jsx

```diff
@@ -0,0 +1,107 @@
+/**
+* SubmissionWarningBanner.jsx
+* Created by Alisa Burdeyny 3/10/17
+*/
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Link } from 'react-router-dom';
+import BannerRow from './BannerRow';
+
+const propTypes = {
+    submissionInfo: PropTypes.object,
+    revertSubmission: PropTypes.func
+};
+
+const defaultProps = {
+    submissionInfo: null,
+    revertSubmission: null
+};
+
+export default class SubmissionWarningBanner extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
```

Comment:
Could we pass down the `reverting` state from `SubmissionStepsContainer`?  If so, we could remove this component state, pass `this.props.revertSubmission` to the button's onClick directly, remove the redundant revert function and make this a functional component.